app/uk/gov/hmrc/excisemovementcontrolsystemapi/repository/MovementRepository.scala

```diff
@@ -92,10 +123,23 @@ object MovementMessageRepository {
           .background(true)
           .unique(true)
       ),
-      IndexModel(
-        Indexes.ascending("administrativeReferenceCode"),
-        IndexOptions().name("arc_index")
-          .background(true)
-      )
+      createIndexWithBackground("administrativeReferenceCode", "arc_index"),
+      createIndex("consignorId", "consignorId_ttl_idx"),
+      createIndex("consigneeId", "consigneeId_ttl_idx")
+    )
+
+  def createIndex(fieldName: String, indexName: String): IndexModel = {
+    IndexModel(
+      Indexes.ascending(fieldName),
+      IndexOptions().name(indexName)
     )
+  }
+
+  def createIndexWithBackground(fieldName: String, indexName: String): IndexModel = {
+    IndexModel(
+      Indexes.ascending(fieldName),
+      IndexOptions().name(indexName)
+        .background(true)
+    )
+  }
```

Comment:
These two methods are pretty much the same but the only difference is the background, could they be refactored into one that takes a background parameter and is defaulted to false? If we call .background(false), is that the same as not calling .background() at all?